data/human/human missions.txt

```diff
@@ -6893,3 +6893,1806 @@ mission "Lost Dog: Homeward Bound"
+			`	She shrugs, and winks, "Anyways, I got plenty of work that needs doing. Take care of yourself, Capt'n."`
+			`	With that she heads back to her freighter fleet.`
+
+	npc accompany save
+		government "Merchant" # 3800 cargo capacity
+		personality escort timid
+		ship "Behemoth" "H.H.H. Gullveig's Revenge"
+		ship "Hauler" "H.H.H. Hard and Fast"
+		ship "Hauler" "H.H.H. Red Rambler"
+		ship "Hauler III" "H.H.H. Heavy Haulage"
+		ship "Hauler III" "H.H.H. Hroar's Heavy Hitter"
+		ship "Hauler II" "H.H.H. First Last Chance"
+		ship "Hauler II" "H.H.H. Lucky Hauler"
+		ship "Mule" "H.H.H. Herald of Vermilion"
+		ship "Bulk Freighter" "H.H.H. Cardinal Conveyor"
+		ship "Bulk Freighter" "H.H.H. Muspel Zipper"
+		ship "Freighter" "H.H.H. Ruby Rails"
+		ship "Freighter" "H.H.H. Dusty Boots"
+
+
+mission "Timothy Radrickson 3d: Return to Rand"
+	landing
+	source "Wayfarer"
+	destination "Rand"
+	to offer
+		has "Timothy Radrickson 3c: Escort the Convoy: done"
+	on complete
+		conversation
+			`Instead of Regina you are met by Timothy's executive secretary, Wynnafrith. She is clearly a little uncomfortable in Rand's regular gravity, but is doing her best to push through it.`
+			`	"Captain <last>! You're the talk of the world! The savior of Rand." She gives you an open and earnest smile.`
+			`	"I've been sent by the Governor to drive you straight to his estate. Please come with me."`
+			choice
+				`	(Go to the Governor's Mansion.)`
+
+			`	In the evening the passing landscape is too dark to see, so instead you focus on the glittering stars overhead. Eventually your ride slows as you seem to be caught up in traffic near the Governor's mansion. As you approach you pass streets crowded with parked personal vehicles.`
+			`	The mansion is lit up with dozens of flood lights, and brimming with fabously dressed people. Wynnafrith turns her vehicle over to a valet and opens your door to let you out.`
+			`	She gives you a mischievous grin. "Apologies, Captain <last>. I forgot to mention the Governor was holding a little get together. Please follow me."`
+			choice
+				`	(Follow Wynnafrith.)`
+			`	She leads you through an army of valets, waiters, and attendants as you weave in and out of the various rooms and halls. Many of the guests appear to recognize you and offer you earnest congratulations. Eventually you are ushered to the table of honor where Wynnafrith introduces you to a half dozen company-bosses, senior executives, political operatives, and the like. You notice Regina here as well, but she is seated at a table in the back, on the fringes of the upper crust of society.`
+			`	Not long after being seated Governor Timothy appears and after making small talk with many of the gathered illuminaries sits beside you, "I have something to admit." He says in quietly to you.`
+			`	He gives you a sheepish smile, "This is a celebration in your honor, Hero of Rand."`
+			choice
+				`	"Hero of Rand?"`
+				`	"What do mean?"`
+				`	"So you lied to me."`
```

Comment:
Maybe provide an option for the player to skip the party if they don't feel like schmoozing with the partygoers?

Reply:
Given how you were driven to and deposited at the mansion, I think you're stuck there until the event concludes.

Reply:
Alright, I have added an option to leave the party and sit by yourself in room instead.